joneva_duration.go

```diff
@@ -0,0 +1,44 @@
+package joneva
+
+import (
+	"fmt"
+	"time"
+)
+
+// Duration получает значение ключа как time.Duration.
+// Поддерживает стандартные форматы Go: "300ms", "1.5h", "2h45m", "30s", "5m" и т.д.
+// Также поддерживает числа как наносекунды.
+func (j *Joneva) Duration(key string) (time.Duration, error) {
+	valueInterface := j.Get(key)
+	if valueInterface == nil {
+		return 0, fmt.Errorf("%v: %w", key, ErrNotFound)
+	}
+
+	switch v := valueInterface.(type) {
+	case string:
+		return time.ParseDuration(v)
+	case float64:
+		// Числа интерпретируются как наносекунды
+		return time.Duration(v), nil
+	case int:
+		return time.Duration(v), nil
+	case int64:
+		return time.Duration(v), nil
```

Comment:
In Go, when parsing JSON, numbers are by default decoded into float64 if the target variable is an interface{} or a generic type. These lines are unreachable code.